### ISSUE TITLE
Incoherent space in Log view

### ISSUE THREAD

**Issue by melomac** [Vector35/binaryninja-api#1674]
At least on macoOS, and depending on the font name and the font size, the following code doesn't show nicely in the Log view:

```python
import numpy as np
import random

def p(t):
    log_info(str(np.array(t).reshape((16, 16))))

# ok
x = range(256)
p(x)

# ko
p(random.sample(x, k=len(x)))
```

| Result | Font name | Font size |
|-|-|-|
| ❌ | Inconsolata | 11
| ✅ | Inconsolata | 12
| ❌ | Inconsolata | 13
| ✅ | Inconsolata | 14
| ❌ | Source Code Pro | 11
| ❌ | Source Code Pro | 12
| ❌ | Source Code Pro | 13
| ❌ | Source Code Pro | 14
| ❌ | SF Mono | 11
| ❌ | SF Mono | 12
| ✅ | SF Mono | 13
| ❌ | SF Mono | 14

SF Mono comes from `/System/Applications/Utilities/Terminal.app/Contents/Resources/Fonts/`.

**Comment by psifertex** [comment]
Still a bug in QT6

**Comment by psifertex** [comment]
<img width="560" alt="Screen Shot 2021-04-14 at 3 35 21 PM" src="https://user-images.githubusercontent.com/140215/114768550-07874600-9d37-11eb-8ffe-0dcaf05ee6c2.png">

**Comment by CouleeApps** [comment]
Fixed in 3.5.4281-dev
